CPAC/pipeline/nipype_pipeline_engine/engine.py

```diff
@@ -72,28 +72,28 @@ def __init__(self, *args, mem_gb=DEFAULT_MEM_GB, **kwargs):
         self.seed = random_seed()
         self.seed_applied = False
         self.input_data_shape = Undefined
-
+        self._mem_x = {}
         if 'mem_x' in kwargs and isinstance(
             kwargs['mem_x'], (tuple, list)
         ):
-            mem_x = {}
             if len(kwargs['mem_x']) == 3:
                 (
-                    mem_x['multiplier'],
-                    mem_x['file'],
-                    mem_x['mode']
+                    self.mem_x['multiplier'],
```

Comment:
@shnizzedy one question I have: above you define `self._mem_x`, but throughout this codeblock you're modifying `self.mem_x`. Could you just clarify for me what the rationale is here? Thanks!

Reply:
Whoops! I refactored that block in https://github.com/FCP-INDI/C-PAC/pull/1701/commits/9bd58de62d34a74b9091b795cf9bed74d934dd5f and that's an error. Those should be `self._mem_x`. Thanks for catching that! I'll fix it now

Reply:
Fixed in e81e7c7

Reply:
I believe that [this patch commit](https://github.com/FCP-INDI/C-PAC/commit/e81e7c7e371dfca7e30e1eb12fbb1c99e6c29a92) was removed from the branch; was that intentional?